package.json

```diff
@@ -30,6 +32,7 @@
     "@babel/preset-react": "^7.22.15",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@juggle/resize-observer": "^3.4.0",
```

Comment:
had to switch our polyfill over from "resize-observer-polyfill" to this as the old one didn't play nicely with ts compilation. Seems to no longer be maintained and this library was recommended by others who came across the same issue. Slots in like for like.

Reply:
https://app.warp.dev/block/98XN9HmSMLgfg51K0iW9zW

I tried installing this in VIRTO.TEST and it threw a bunch of errors in during build. 

Not sure how this worked originally because the old dependency was not a peerDependency. I suspect that we were getting lucky and one of our other common dependencies also relied on that polyfill so it technically was in our node_modules.